addons/web/static/tests/core/action_swiper.test.js

```diff
@@ -141,7 +164,7 @@ test("can perform actions by swiping to the right", async () => {
         },
     });
     await dragHelper.drop();
-    await animationFrame();
+    await advanceTime(500);
```

Comment:
Why 500 ? And could we extract it to a constant/reuse the ActionSwiper's value ?